.github/workflows/build-deps.yml

```diff
@@ -98,3 +98,42 @@ jobs:
     - name: Publish image
       if: github.event_name == 'push' && steps.changed-files.outputs.builders_any_changed == 'true'
       run: docker push ghcr.io/datadog/agent-int-builder:${{ matrix.job.image }}
+
+  build-deps-macos:
+    name: Target macOS
+    runs-on: macos-12
+    steps:
+      - name: Set up environment
+        run: |
+          # We remove everything that comes pre-installed via brew to avoid depending or shipping stuff that
+          # comes in the runner through brew to better control what might get shipped in the wheels via `delocate`
+          brew remove --force --ignore-dependencies $(brew list --formula)
+          brew install coreutils
+
+      - name: Set up Python
+        env:
+          # Despite the name, this is built for the macOS 11 SDK on arm64 and 10.9+ on intel
```

Comment:
How is this running?

Reply:
I don't understand the question.

Reply:
As in how is the binary running successfully if it was built for a different architecture?

Reply:
Universal2 binaries are compatible with both Intel and arm64 (https://en.wikipedia.org/wiki/Universal_binary), the comment means to explain that the 11 in the package name refers to the SDK version used for the arm64 variant, whereas for intel (which is the one we're actually using in the build) it is built with 10.9, meaning we should be able to build wheels with support for MacOS versions 10.9+.